execute_pipeline.py

```diff
@@ -106,22 +114,53 @@ def _parse_args():
 
   pipeline_args = {}
   repl_vars = {"REPOROOT": flags.reporoot}
+  configs = []
+  pipeline_name = flags.pipeline_name
+  if flags.preset:
+    (configs, pipeline_name) = _parse_preset(flags.preset)
   if flags.config:
-    for config_spec in flags.config.split(','):
-      config_args = _load_config_spec(config_spec, repl_vars)
-      pipeline_args.update(config_args)
+    configs = configs + flags.config.strip().split(',')
+  for config_spec in configs:
+    config_args = _load_config_spec(config_spec, repl_vars)
+    pipeline_args.update(config_args)
 
   cmd_args = ast.literal_eval(flags.pipeline_kwargs)
   pipeline_args.update(cmd_args)
   print "Final args:"
   for (k, v) in pipeline_args.iteritems():
     print " ", k, ":", v
 
-  return (flags.pipeline_name,
+  return (pipeline_name,
           pipeline_args,
           flags.remote_mode)
 
 
+def _parse_preset(preset):
+    (api_name, language, task) = preset.strip().split('-')
+    configs = ["../googleapis/gapic/api/artman_%s.yaml:common|%s" % (api_name, language),
+        "../googleapis/gapic/lang/common.yaml:default|%s" % language]
```

Comment:
Knowledge of googleapis is laid out and which pipelines exist should not be in the execute_pipelines script. It should be accessed in an abstract way by execute_pipelines, and encoded next to the actual pipelines.